data/hai/hai outfits.txt

```diff
@@ -775,7 +775,7 @@ outfit "Pulse Rifle"
 outfit "Shield Beetle Pendant"
 	category "Unique"
 	thumbnail "outfit/crystal pendant"
-	"quantum keystone" 1
+	"inertia reduction" 0.05
```

Comment:
```suggestion
	"inertia reduction" 0.01
```
5% is what I gave to Quarg engines in the other PR... I don't think it should be that noticeable at all
Not deadset on this, its fine for me either way, since I don't care about lore much but for 1% we can just say it's the crew being inspired by this.

Reply:
On the one hand, this is a single one-off outfit. The player presumably can't acquire any additional ones, so we don't need to worry about OP stacking. But it also means that we can't set it so low that it is basically unnoticeable unless they have a bunch of them.

I'd personally prefer if this is locked to the protagonist, so no shuffling around ships, but in the meantime, even a 5% is an easily achievable amount of extra performance that many competent mechanics and tuners can get out of engines or skill in handling. Likewise, we could construe that either a) crew are inspired by the pendant, or b) the location of the pendant represents where the protagonist is focusing their inspirational or tuning efforts. I rather like that it doesn't actually *specify* what is going on. So we can assume it is the protagonist's personal skill, or them inspiring their people to do better, or some combination thereof, and the player can read any combination of those into things.

Reply:
Yeah I'd be tempted too tbh
Okay fair 5% lets see what others think. I was just thinking content and lore wise (which is out of my zone of comfort so idk)